src/java/bobbybot/Event.java

```diff
@@ -0,0 +1,43 @@
+package bobbybot;
+
+/** Represents a bobbybot.Task that should be completed at a specified time period
+ *  @author mokdarren
+ */
+public class Event extends Task {
+    protected String at;
+
+    /**
+     *
+     * @param description description of task
+     * @param at time period of bobbybot.Event (start-end)
+     */
+    public Event(String description, String at)  {
```

Comment:
Same comment as the usage of `by`. Perhaps using `eventTiming` would be better than `at`.